css/styles.css

```diff
@@ -0,0 +1,524 @@
+    content: "\f105";
+    font-family: "FontAwesome";
+    margin-left: -10px;
+    opacity: 0;
+    vertical-align: middle;
+    transition: margin .3s, opacity .3s;
+}
+
+.work-card .description .read-more a:hover:after {
+    margin-left: 5px;
+    opacity: 1;
+}
+
+.work-card p {
+    position: relative;
+    margin: 1rem 0 0;
+}
+
+.work-card p:first-of-type {
+    margin-top: 2rem;
+}
+
+.work-card p:first-of-type:before {
+    content: "";
+    position: absolute;
+    height: 5px;
+    background: #FC5956;
+    width: 35px;
+    top: -0.75rem;
+    border-radius: 3px;
+}
+
+@media only screen and (min-width: 640px) {
+    .work-card {
+        -webkit-box-orient: horizontal;
+        -webkit-box-direction: normal;
+            -ms-flex-direction: row;
+                flex-direction: row;
+        max-width: 700px;
+    }
+    .work-card .meta {
+        -ms-flex-preferred-size: 40%;
+            flex-basis: 40%;
+        height: auto;
+    }
+    .work-card .description {
+        -ms-flex-preferred-size: 60%;
+            flex-basis: 60%;
+    }
+    .work-card .description:before {
+        -webkit-transform: skewX(-3deg);
+                transform: skewX(-3deg);
+        content: "";
+        background: #fff;
+        width: 30px;
+        position: absolute;
+        left: -10px;
+        top: 0;
+        bottom: 0;
+        z-index: -1;
+    }
+    .work-card.alt {
+        -webkit-box-orient: horizontal;
+        -webkit-box-direction: reverse;
+            -ms-flex-direction: row-reverse;
+                flex-direction: row-reverse;
+    }
+    .work-card.alt .description::before {
+        left: inherit;
+        right: -10px;
+        -webkit-transform: skew(3deg);
+                transform: skew(3deg);
+    }
+}
+
+
+/* ABOUT */
+
+.about {
+    background: #F2C041; 
+    padding-bottom: 8vh;
+}
+.about .details {
+    width: 100%;
+}
+
+.about .details h1 {
+    margin: 0 auto;
+    color: #FC5956;
+    padding: 10vh 8vh 4vh 8vh;
+}
+
+.about .details h2 {
+    font-size: 2rem;
+}
+
+.about .details p {
+    margin: 0 8vh 4vh 8vh;
+}
+
+.about .details .dot {
+    color: #1C1B1F; 
+}
+
+.about .img .texas {
+    height: 200px;
+    margin: 0 auto;
+}
+
+ @media only screen and (min-width: 767px) {
+    .about {
+        background: #F8F8F8;
+        clear: both;
+        overflow: hidden;
+        padding-bottom: 0;
+    }
+    .about .details {
+       width: 50%;
+       float: left;
+    }
+    .about .img {
+        width: 50%;
+        float: right;
+        background: #F2C041;
+        height: 100%;
+    }
+    .about .img .texas {
+        height: 500px;
+        padding: 8vh;
+    }
+ }
+
+ 
+ /* CONTACT */
+
+.contact .title h2 {
+    color: #FC5956;
+    padding-left: 4vh;
+    margin-bottom: 2vh;
+}
+
+.contact .title h1 {
+    padding-left: 4vh;
+    margin: 0 auto 4vh 0;
+}
+
+.contact .title .dot {
+    color: #F2C041;
+}
+
+.contact .contact-details address p .highlight {
```

Comment:
careful with selector over specificity, this will cause headaches on bigger projects, general rule of thumb is 3 selectors should be enough, 4 if you HAVE to, never more